### PR TITLE
Rover

### PR DESCRIPTION
**Pull request by yegor256**
Implemented `Rover` rule from the paper

## Summary by CodeRabbit

- **New Features**
  - Introduced a new normalization rule for the "over" operation.
- **Tests**
  - Added multiple test cases to validate the normalization behavior of the "over" rule, covering various input patterns and edge cases.
- **Documentation**
  - Improved formatting and readability of normalization rule outputs in test resources.

### CODE REVIEW

**Comment by yegor256** [comment]
@maxonfjvipon should be good to go, please check

**Comment by yegor256** [comment]
@maxonfjvipon wait, it's not ready yet

**Review by maxonfjvipon — APPROVED** [review]

File: resources/over.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 Objectionary.com
+# SPDX-License-Identifier: MIT
+---
+name: OVER
+pattern: '⟦!B1, !a ↦ !e1, !B2⟧(!a ↦ !e2)'
+result: ⊥
+where:
+  and:
+    - not:
+        - eq:
```
Comment (maxonfjvipon) [inline resources/over.yaml:10]: It should be just `eq:`, without dash.

File: test-resources/rewriter-packs/normalize/over-with-rho.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 Objectionary.com
+# SPDX-License-Identifier: MIT
+---
+# yamllint disable rule:line-length
+rules:
+  basic: ['over']
+input: |
+  Φ ↦ ⟦ ρ ↦ ξ.t, k ↦ ∅ ⟧ (ρ ↦ ξ.k)
+output: |-
+  Φ ↦ ⟦ ρ ↦ ξ.k, k ↦ ∅ ⟧
```
[the remark this answers is not in the document]
Comment (yegor256) [reply test-resources/rewriter-packs/normalize/over-with-rho.yaml:10]: @maxonfjvipon it does work, but incorrectly:

```
  1) Rewriter, rewrite packs, normalize/over-with-rho.yaml
       expected: "Φ ↦ ⟦ ρ ↦ ξ.k, k ↦ ∅ ⟧ (ρ ↦ ξ.k)"
        but got: "Φ ↦ ⊥"
```

**Comment by yegor256** [comment]
@maxonfjvipon it's ready for review

**Comment by 0crat** [comment]
@maxonfjvipon Thanks for the review! You've earned +4 points for this: +12 as a basis; -10 for very few (5) comments; +2 to give you at least something.  Your running balance is [-188](https://www.eolang.org/zerocracy/objectionary-vitals.html).

**Comment by 0crat** [comment]
@yegor256 Hey there, awesome job on your contribution! 🎉 You've snagged +21 points: 16 for showing up and an extra 5 for those 102 [hits-of-code](https://www.yegor256.com/2014/11/14/hits-of-code.html) you cranked out. That's a solid effort without going overboard (no deductions for you!). Keep this momentum going, and you'll be racking up points in no time. Remember, more reviews and fewer comments can boost your score even higher. Your current balance sits at [-216](https://www.eolang.org/zerocracy/objectionary-vitals.html), but don't let that discourage you. [Keep](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) those contributions flowing, and you'll be in the green before you know it!
